indexer/editable_map_object.cpp

```diff
@@ -94,12 +96,34 @@ void EditableMapObject::SetNearbyStreets(vector<string> && streets)
 {
   m_nearbyStreets = move(streets);
 }
-void EditableMapObject::SetHouseNumber(string const & houseNumber)
+
+bool EditableMapObject::SetHouseNumber(string const & houseNumber)
 {
-  // TODO(AlexZ): Check house number for validity with feature::IsHouseNumber ?
+  // TODO: Improve this basic limit.
+  auto constexpr kMaxHouseNumberLength = 25;
+  if (m_houseNumber.size() > kMaxHouseNumberLength)
+    return false;
+
+  bool hasDigits = false;
+  // TODO: Should we allow arabic numbers like U+0661 ١	Arabic-Indic Digit One?
+  for (auto c : m_houseNumber)
```

Comment:
Good catch! Сейчас поправлю.

> On Apr 6, 2016, at 19:41, Dmitry Yunitsky <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.cpp <https://github.com/mapsme/omim/pull/2802#discussion_r58739628>:
> 
> >  {
> > -  // TODO(AlexZ): Check house number for validity with feature::IsHouseNumber ?
> > +  // TODO: Improve this basic limit.
> > +  auto constexpr kMaxHouseNumberLength = 25;
> > +  if (m_houseNumber.size() > kMaxHouseNumberLength)
> > +    return false;
> > +
> > +  bool hasDigits = false;
> > +  // TODO: Should we allow arabic numbers like U+0661 ١	Arabic-Indic Digit One?
> > +  for (auto c : m_houseNumber)
> Пустые номера домов будут невалидны?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2802/files/28c906af0d4709e796dd4fb05bf395153131032c#r58739628>